app/forms/sufia/forms/permission_template_form.rb

```diff
@@ -26,7 +26,7 @@ def update(attributes)
 
         # This allows the attributes
         def grants_as_collection(attributes)
-          attributes_collection = attributes[:access_grants_attributes]
+          attributes_collection = Array.wrap(attributes[:access_grants_attributes])
```

Comment:
I see what you mean. Thanks for pointing that out.  I'll replace this with something like:
```
attributes_collection = attributes.include?(:access_grants_attributes) ? attributes[:access_grants_attributes] : []
```